Dockerfile

```diff
@@ -68,10 +69,12 @@ EXPOSE 8000
 
 # Install operating system dependencies.
 RUN apt-get update --yes --quiet && apt-get install --yes --quiet --no-install-recommends \
+    ca-certificates \
     build-essential \
     libpq-dev \
     curl \
     git \
+    gnupg \
```

Comment:
Those are necessary packages to use this installer.

Reply:
question: If you only need these in the dev container, would it make sense to move them in to the `apt-get` run in the dev stage (around line 140)?

Reply:
(although just realised you'd need to run `apt-get` twice anyway, but probably still worth limiting dependencies to where they are required)

Reply:
Done